lib/calendar_assistant/cli/repl.rb

```diff
@@ -0,0 +1,70 @@
+require 'readline'
+
+class CalendarAssistant
+  module CLI
+    class Repl
+      class History
+        HISTORY_FILE = '~/.thor_interactive_history'
```

Comment:
Could we put this in a file named `.calendar-assistant.history` or something similar? Trying to match the files we already create.

And for that matter, maybe we should put all files in a subdirectory ... see #111

Reply:
Yep makes sense, I had it set up this way so that it could become a gem - but I think one step at a time makes sense here.